### DISCUSSION TITLE
Keep / remember tabs when DBeaver is closed

### DISCUSSION THREAD

**Discussion by clounie** [dbeaver/dbeaver#13428] [Q&A]
**Is your feature request related to a problem? Please describe.**
Every time I close DBeaver I have to save or lose tabs.

This is very annoying, and is my only complaint with DBeaver.

**Describe the solution you'd like**
DBeaver should instead just remember tabs without making me save them (Sublime Text is a great example of this)

**Describe alternatives you've considered**
One alternative might be to save a "set of tabs" which you can reload on DBeaver starting up - but that solution is not even close to ideal.

**Additional context**
None

**Comment by cerebel** [comment]
@clounie #2496

**Comment by clounie** [comment]
I have that option checked, but unless I 'save' my editors, it re-opens them...and they're complete empty of any content.

The point of this issue is that I don't _want_ to have to save anything. Either:
1. Saving the tab saves to a file in the file system 
  * ...then this is a feature request - now remember my content without making me save anything
2. Saving the tab does not save to a file but to some DBeaver-specific location 
 * ...then this is a "fix bad UX" request. Just remember the tab next time DBeaver starts up, and don't make me press save. (If I wanted it _not_ to save, I would reference the option you linked above)

**Comment by uslss** [comment]
1. Reopen database editors after restart option should be checked.
![image](https://user-images.githubusercontent.com/31996417/128055773-89bebb31-b36d-4b31-90aa-621531758843.png)
2. Autosave after any modification option should be checked.
![image](https://user-images.githubusercontent.com/31996417/128055984-e2eb4836-d7d4-4221-aad2-220a19da12ff.png)
